jsonrpc/eth_endpoint.go

```diff
@@ -547,3 +547,6 @@ func (e *Eth) Unsubscribe(id string) (bool, error) {
 	ok := e.d.filterManager.Uninstall(id)
 	return ok, nil
 }
+func createBlockNumberPointer(x BlockNumber) *BlockNumber {
+	return &x
+}
```

Comment:
Please remove this method from the `eth_endpoints.go` file as this file is exclusively meant to house the dispatch endpoints.

You can move it to a more appropriate place (for example `codec.go`, where the constant block number values reside)

Reply:
I would go even as far as to refactor this method to only support the constant block numbers that we have defined in `codec.go` as we don't use it for anything other than getting the reference to these constants.

Reply:
Sure Milos, will do the required refactoring.

Reply:
@zivkovicmilos  this method can also be used to  return a reference of  non-constant blockNumber value, so i am just moving this into  `codec.go` for now.

Reply:
Why would we need a separate function to get a reference to a non-constant `BlockNumber` type?

Reply:
@munna0908 tagging for visibility

Reply:
@lazartravica Thanks for tagging, will close it ASAP

Reply:
@zivkovicmilos Fixed it Commit-[link](https://github.com/0xPolygon/polygon-sdk/pull/148/commits/37f93254e322b6c583fb816435ad0b0f5563baae)

Reply:
Looks good 👍